Loungeware/scripts/microgame_metadata/microgame_metadata.gml

```diff
@@ -0,0 +1,336 @@
+function ___init_metadata(){
```

Comment:
This file is deleted and shouldn't be in here. Are you sure the merge went okay? Are there any missing folders or files?